### ISSUE TITLE
Problems with provision

### ISSUE THREAD

**Issue by joelgoop** [demokratikollen/demokratikollen#15]
I'm having problems with provisioning on the `develop` branch. For some reason it is trying to install Flask to what seems to be the system python 2.7 directory and of course since this should be in a virtualenv and is run without `sudo`, it fails. I'm not sure since what commit I have this problem. Is it only me? What do you think @joharohl? 

The following is an excerpt from the end of .pip/pip.log:

```
creating build/lib.linux-i686-2.7/flask/testsuite/test_apps/subdomaintestmodule/static

copying flask/testsuite/test_apps/subdomaintestmodule/static/hello.txt -> build/lib.linux-i686-2.7/flask/testsuite/test_apps/subdomaintestmodule/static

running install_lib

creating /usr/local/lib/python2.7/dist-packages/flask

error: could not create '/usr/local/lib/python2.7/dist-packages/flask': Permission denied

----------------------------------------
Cleaning up...
  Removing temporary dir /tmp/pip_build_vagrant...
Command /usr/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip_build_vagrant/Flask/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-lWrK6t-record/install-record.txt --single-version-externally-managed --compile failed with error code 1 in /tmp/pip_build_vagrant/Flask
Exception information:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/pip-1.5.6-py2.7.egg/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/usr/local/lib/python2.7/dist-packages/pip-1.5.6-py2.7.egg/pip/commands/install.py", line 283, in run
    requirement_set.install(install_options, global_options, root=options.root_path)
  File "/usr/local/lib/python2.7/dist-packages/pip-1.5.6-py2.7.egg/pip/req.py", line 1435, in install
    requirement.install(install_options, global_options, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pip-1.5.6-py2.7.egg/pip/req.py", line 706, in install
    cwd=self.source_dir, filter_stdout=self._filter_install, show_stdout=False)
  File "/usr/local/lib/python2.7/dist-packages/pip-1.5.6-py2.7.egg/pip/util.py", line 697, in call_subprocess
    % (command_desc, proc.returncode, cwd))
InstallationError: Command /usr/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip_build_vagrant/Flask/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-lWrK6t-record/install-record.txt --single-version-externally-managed --compile failed with error code 1 in /tmp/pip_build_vagrant/Flask
```

**Comment by joelgoop** [comment]
When start by running `vagrant up` on the `master` branch (which works) and then checkout `develop` and run `vagrant provision` I get no errors, but as soon as I start from scratch on `develop` (after `vagrant destroy`) it fails.

**Comment by joelgoop** [comment]
Ok, I think I have identified the error: The ansible check if `pip` is installed detects the global `pip` and doesn't install `pip` into the `virtualenv`. Therefore requirements are installed globally using `pip` for Python 2.7 and fails since it requires `sudo`. I am attempting to fix it.

**Comment by joharohl** [comment]
Hmm. I thought I tested this with a fresh `vagrant up`. I can have a look if you want to do something else.

**Comment by joelgoop** [comment]
I'm testing my quick fix now. If it seems to work I'll push in a few minutes, then you can have a look at it and see if it needs improving. I'll post again if it doesn't work.

**Comment by joelgoop** [comment]
Closed by ec160a2db87f753580a54bcce02af6dd902cc503. Feel free to test it.

**Comment by joharohl** [comment]
That is the way I would have changed it as well. Strangely the old version works for me. I wonder if this is caused by the windows work around.
